ehrql/main.py

```diff
@@ -306,7 +306,7 @@ def generate_measures_with_dummy_data(
         reader = read_rows(dummy_data_file, column_specs)
         results = iter(reader)
     elif dummy_tables_path:
-        log.info(f"Reading CSV data from {dummy_tables_path}")
+        log.info(f"Reading data from {dummy_tables_path}")
```

Comment:
Nit: these two log changes are different, which is surprising.

Reply:
Ah, you're quite right. However looking at this more closely, I think the problem was that they shouldn't have been identical beforehand because they do different things.